less/common/Search.less

```diff
@@ -59,7 +59,7 @@
     width: 225px;
     padding-left: 32px;
     padding-right: 32px;
-    .transition(all 0.4s);
+    transition: all 0.4s;
```

Comment:
We should probably avoid `all`. It's not your doing, but I wouldn't be opposed to you changing it ;)

https://www.joshwcomeau.com/animation/css-transitions/